src/utils/redirects.test.js

```diff
@@ -3,41 +3,41 @@ const { processRedirectString, getRedirect } = require('./redirects')
 
 describe('processRedirectString', () => {
   it('reads the regex, replacement and code', () => {
-    const { regex, replace, code } = processRedirectString('^/foo /bar 418')
+    const { regex, replace, code } = processRedirectString('^/foo /bar 308')
 
     expect(regex).toBeInstanceOf(RegExp)
     expect(regex.source).toEqual('^\\/foo')
     expect(replace).toEqual('/bar')
-    expect(code).toEqual(418)
+    expect(code).toEqual(308)
```

Comment:
You removed my teapot joke :( it's old anyway.

Reply:
Wow I didn't even remember that one. I thought it was just a random number haha. Reverted now.